pyaedt/modeler/PrimitivesNexxim.py

```diff
@@ -79,6 +80,106 @@ def components_catalog(self):
             self._components_catalog = ComponentCatalog(self)
         return self._components_catalog
 
+    @pyaedt_function_handler()
+    def create_subcircuit(self, location=None, angle=0, name=None, nested_subcircuit_id=None):
+        """Add a new Circuit subcircuit to the design.
+
+        Parameters
+        ----------
+        location : list of float, optional
+            Position on the X axis and Y axis.
+        angle : float, optional
+            Angle rotation in degrees. The default is ``0``.
+        name : str, optional
+            Name of the Design. The default is ``None``, in which case
+            a unique name will be generated.
+        nested_subcircuit_id : str, optional
+            Id of nested subcircuit.
+            Example `"U1"`.
+
+        Returns
+        -------
+        :class:`pyaedt.modeler.Object3d.CircuitComponent`
+            Circuit Component Object when successful or False when fails.
+
+        Examples
+        --------
+        >>> from pyaedt import Circuit
+        >>> cir = Circuit()
+        """
+        if not name:
+            name = generate_unique_name("Circuit")
+
+        if nested_subcircuit_id:
+            parent_name = "{}:{}:{}".format(
+                self._app.design_name.split("/")[0], nested_subcircuit_id, os.urandom(1, 10000)
```

Comment:
urandom only tke one argument. I am assuming here a low int would do it as it for example:
```suggestion
                self._app.design_name.split("/")[0], nested_subcircuit_id, os.urandom(5)
```